README.md

```diff
@@ -91,7 +117,15 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+
+var doSomething = function(thingToDo) {
+  thingToDo();
+};
+
+var thingToDo = function() {
+  console.log("Do something!");
+};
+
 ```
```

Comment:
`thingToDo` is the argument for `doSomething`, which when called should be replaced with an actual function. Here's an example of `doSomething` being called, with an anonymous function being passed in as an argument.

```js
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```